src/target_intacct/client.py

```diff
@@ -229,10 +229,18 @@ def format_and_send_request(self, data: Dict) -> Union[List, Dict]:
         Returns:
             A response from the _post_request (dict).
         """
-
         key = next(iter(data))
-        object_type = data[key]["object"]
 
+        if use_key:
+            object_type = data[key]["object"]
+           
```

Comment:
Consider deleting so the formatting of the two assignment blocks is identical, which makes the two branches easier to compare.